cvat/apps/dataset_manager/serializers.py

```diff
@@ -10,6 +10,7 @@ class DatasetFormatSerializer(serializers.Serializer):
     ext = serializers.CharField(max_length=64, source='EXT')
     version = serializers.CharField(max_length=64, source='VERSION')
     enabled = serializers.BooleanField(source='ENABLED')
+    dimension = serializers.CharField(max_length=64, source='DIMENSION')
```

Comment:
Why do you need 64 chars here? It is better to have choice here between 2D and 3D.

Reply:
Yes we will change it to have choices.

Reply:
Made changes accordingly.